ZupIT/ritchie-cli
# Create journey tips

**- What I did**
closes https://github.com/ZupIT/ritchie-cli/issues/289

Created a tutorial command  (this command enables and disables the tutorial) and created the tutorial tips, the first tip is `rit init` then just follow the tutorial's own tips.

>The defined journey:
![Ritchie's journey flow](https://user-images.githubusercontent.com/67295691/88417505-688d7900-cdb8-11ea-82ae-31783d780adf.png)


>Tutorial command:
![command-tutorial](https://user-images.githubusercontent.com/14079465/88421444-f9ffe980-cdbe-11ea-8eef-13b1892b1d87.gif)

>Journey:
![jornada](https://user-images.githubusercontent.com/14079465/88436773-62f55a80-cddb-11ea-9592-cd912a23e80c.gif)
_Updated 24 jul at 18:28_

Tutorial tip for rit init that appears only if rit init needs to be run.
![tutorial-rit-e-ritinit](https://user-images.githubusercontent.com/14079465/88421438-f8cebc80-cdbe-11ea-84dc-e0e19e805647.gif)


**- How to verify it**
```bash
rit tutorial # Enable and disable the tutorial and you can see the status of the tutorial saved in: `.rit / tutorial`
rit init # View a tutorial tip (when the tutorial is enabled)
```


**- Description for the changelog**


Added tutorial functionality with hints and tutorial command
